bundle/config/validate/enum.go

```diff
@@ -0,0 +1,104 @@
+package validate
+
+import (
+	"context"
+	"fmt"
+	"slices"
+	"sort"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/internal/validation/generated"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/dyn"
+)
+
+type enum struct{}
+
+func Enum() bundle.Mutator {
+	return &enum{}
+}
+
+func (f *enum) Name() string {
+	return "validate:enum"
+}
+
+func (f *enum) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
+	diags := diag.Diagnostics{}
+
+	// Generate prefix tree for all enum fields.
```

Comment:
Curious how much time does it take. Can we add debug message with stats? "Prefix tree with 125 nodes generated in 125ms"